libs/design-system/nav-header/src/lib/nav-header.component.ts

```diff
@@ -0,0 +1,30 @@
+import { CommonModule } from '@angular/common';
+import { ChangeDetectionStrategy, Component, input } from '@angular/core';
+import { MatButtonModule } from '@angular/material/button';
+import { MatIconModule } from '@angular/material/icon';
+import { IconButtonSizeDirective } from '@hra-ui/design-system/icon-button';
+import { AppLogosComponent, AppLogosVariant } from '../../../logos/apps/src/index';
```

Comment:
Need to update the location and tsconfig.base.json of app logos so it can be imported from `@hra-ui/design-system/app-logos` instead